projects/element-ng/tabs-next/si-tabset-next.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="tabs-container">
+  <div
+    #tabContainer
+    class="tab-container-buttonbar-list nav nav-tabs"
+    (siResizeObserver)="resize($event)"
+    (wheel)="mouseScroll($event)"
```

Comment:
do we still need this? Since we using browser scroll I thought this would also be gone